lemminx-liberty/src/main/java/io/openliberty/tools/langserver/lemminx/util/LibertyUtils.java

```diff
@@ -73,6 +74,9 @@ public static boolean isConfigXMLFile(String filePath) {
 
         return isServerXMLFile(filePath) || isConfigDirFile(filePath) ||
                 LibertyProjectsManager.getInstance().getWorkspaceFolder(filePath).hasConfigFile(filePath);
+
```

Comment:
just a comment for going forward...I think we may still want to automatically consider any file that returns true for `isConfigDirFile` or the `src/main/liberty/config/server.xml` as a Liberty config file and apply our xsd to it...even if it does not have a root element of `server`...because it would cause a failure in Liberty server startup if it didn't have a `server` root element.

Reply:
yes agreed. in a local change, I made the isConfigXmlFile logic to be 
```
return isServerXMLFile(filePath) || isConfigDirFile(filePath) || XmlReader.hasServerRoot(new File(filePath))
```